From c821415cbc311daab433e16def9f6c839c71a9f6 Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Sun, 23 Oct 2022 13:35:34 +0000
Subject: [PATCH] Remove Duplicated ro.adb.secure

---
 core/main.mk | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/core/main.mk b/core/main.mk
index 889784280..33e689968 100644
--- a/core/main.mk
+++ b/core/main.mk
@@ -375,13 +375,8 @@ enable_target_debugging := true
 tags_to_install :=
 ifneq (,$(user_variant))
   # Target is secure in user builds.
-  ADDITIONAL_SYSTEM_PROPERTIES += ro.secure=1
   ADDITIONAL_SYSTEM_PROPERTIES += security.perf_harden=1
 
-  ifeq ($(user_variant),user)
-    ADDITIONAL_SYSTEM_PROPERTIES += ro.adb.secure=1
-  endif
-
   ifeq ($(user_variant),userdebug)
     # Pick up some extra useful tools
     tags_to_install += debug
-- 
2.34.1

